src/V2/Records/ListRelatedQuery.php

```diff
@@ -78,7 +73,7 @@ public function setRelatedList(string $relatedListName): void
      *
      * @return $this
      */
-    public function setRelatedListRecord(string $recordId, string $relatedListName)
+    public function setRecordAndRelatedList(string $recordId, string $relatedListName)
```

Comment:
I don't think that this method is very useful. I would prefer you to remove it and use the 2 separate methods that you defined.

Reply:
okkk you're right, it's better